extras/README.md

```diff
@@ -1,16 +1,97 @@
-# New Actor Template
-Use cargo generate to create a new actor module from this template.
+## How to use  built-in `wasmcloud:extras` capability provider
 
-[Learn how to write](https://wascc.dev/tutorials/first-actor/) waSCC actors, or [explore the actor concept](https://wascc.dev/docs/concepts/actors/).
+`wasmcloud` host runtime comes with some built-in capability providers and `wasmcloud:extras` is one of them. In this tutorial we will generate a random number and retrive some information like request id and number of requests using `wasmcloud:extras`. 
 
-To generate new keys, use `make keys`. This will use `nk` to generate both sets of keys for you, and then write them to the `.keys` directory.
+1. Please install `wasmcloud` and `wash` binaries if you are not already did:
```

Comment:
"if you are not already did" should be "if you do not have them installed already"